MarmoUI-Chrome/manifest.json

```diff
@@ -2,17 +2,18 @@
   "manifest_version": 2,
 
   "name": "MarmoUI",
-  "description": "Marmoset Improved! Better UI and functionality",
+  "description": "test Marmoset Improved! Better UI and functionality",
   "version": "1.5",
   "icons": {
     "16": "image/icon16.png",
     "48": "image/icon48.png",
-    "128": "image/icon128.png" 
+    "128": "image/icon128.png"
   },
-  
+
   "content_scripts": [
     {
-      "matches": ["https://marmoset.student.cs.uwaterloo.ca/*"],
+      "matches": ["https://*.uwaterloo.ca/*"],
```

Comment:
I tried that originally, unfortunately it doesn't look like chrome supports that - I'm looking at the [match rules](https://developer.chrome.com/extensions/match_patterns) for reference.

Chrome does report can access anything on *.uwaterloo.ca. Agreed that could cause confusion. 

<img width="460" alt="screen shot 2017-02-06 at 5 25 04 pm" src="https://cloud.githubusercontent.com/assets/1267052/22669130/4933924e-ec91-11e6-820b-8ce618a3bfae.png">